ImGuiFileDialog.cpp

```diff
@@ -3758,7 +3758,7 @@ bool IGFD::FileDialog::Display(const std::string& vKey, ImGuiWindowFlags vFlags,
         }
         m_FileDialogInternal.lastImGuiFrameCount = g.FrameCount;  // mark this instance as used this frame
 
-        m_CurrentDisplayedFlags = ImGuiWindowFlags_None;
```

Comment:
`m_FileDialogInternal.name` is set to `name` in the initial call to the function `Display` as you can see here.